democracy_club/apps/core/report_helpers/views.py

```diff
@@ -22,4 +22,7 @@ def get_context_data(self, **kwargs):
         html = md.convert(rendered_template)
         context["html_content"] = html
         context["toc"] = md.toc
+        context["page_title"] = (
```

Comment:
I've just had a bit of a poke about with this, and found there's `md.toc_tokens`: a list of dicts of the TOC, rather than HTML.

I think replacing this with `md.toc_tokens[0]['name']` produces the same result, without having to parse the HTML